chipy_org/libs/tests/test_nh3_templatetags.py

```diff
@@ -0,0 +1,41 @@
+from django.template import Context, Template
+from django.test import TestCase
```

Comment:
We do have pytest django is installed, if you don't want to rock the old school unit tests framework.  I don't have a strong preference though.  I am just happy to see some tests. 😊